src/Log/Helpers/Assets.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Give\Log\Helpers;
```

Comment:
Not sure this would constitute as a "helper". It's really just the class for handling Assets. Helpers are more like utilities for multiple usage.